truss-chains/truss_chains/utils.py

```diff
@@ -128,6 +130,25 @@ def get_free_port() -> int:
         return port
 
 
+def override_chainlet_to_service_metadata(
+    chainlet_to_service: Mapping[str, definitions.ServiceDescriptor],
+) -> Mapping[str, definitions.ServiceDescriptor]:
```

Comment:
we're ok using the `Mapping` type here since we're not actually mutating the `chainlet_to_service` mapping, we're just modifying its `ServiceDescriptor` contents